starlite/middleware/__init__.py

```diff
@@ -0,0 +1,6 @@
+from starlite.middleware.exceptions import ExceptionHandlerMiddleware
+from starlite.middleware.authentication import AuthenticationResult, AbstractAuthenticationMiddleware
+from starlite.middleware.gzip import GZipMiddleware
+
+
+__all__ = ["ExceptionHandlerMiddleware", "AuthenticationResult", "AbstractAuthenticationMiddleware", "GZipMiddleware"]
```

Comment:
Not sure we need gzip here, since our usage was not meant for user consumption

Reply:
I agree after sleeping on it.  I'm not sure why I thought it was a good idea to include it there last night.